inc/field.class.php

```diff
@@ -218,17 +194,32 @@ public function getFieldTypeName() {
       return strtolower($matches[1]);
    }
 
-   public function getUsedParameters() {
+   public function getEmptyParameters() {
       return [];
    }
 
+   public final function getParameters() {
+      $parameters = $this->getEmptyParameters();
+      foreach ($parameters as $fieldname => $parameter) {
+         $parameter->getFromDBByCrit([
+            'plugin_formcreator_questions_id'   => $this->fields['id'],
+            'fieldname'                         => $fieldname,
+         ]);
+         if ($parameter->isNewItem()) {
+            $parameter->getEmpty();
+         }
+      }
+
+      return $parameters;
+   }
```

Comment:
this method factorizes the foreach loop found in many fields and question classes

Reply:
and also calls getEmpty if the row is mising in the DB